client/dist/style.css

```diff
@@ -0,0 +1,136 @@
+body{
+  background-color: #041c2c;
+}
+
+.carousel_container{
+  overflow-x: hidden;
+  white-space: nowrap;
+  margin-bottom: 20px;
+}
+
+#personnel_carousel, #movies_carousel{
+  width: fit-content;
+  height: 260px;
+  transition: transform .5s ease;
+  white-space: nowrap;
+}
+
+#movies_carousel{
+  height: 206px;
+}
+
+#p_backward, #m_backward{
+  background-image: url(https://www.vudu.com/content/images/pov_left_arrow.svg);
```

Comment:
You will probably want to host this on s3 or whatever media hosting service you end up using.

Reply:
@HBMattoon Good catch - I'll migrate the assets over to S3 asap.